test/fixtures/output/elixir/httpoison/custom-method.ex

```diff
@@ -0,0 +1 @@
+HTTPoison.request(:propfind, "http://mockbin.com/har", "")
```

Comment:
Does HTTPPoison recognize `:propfind` here?

Reply:
Yes, this is a generic request function, e.g. https://hexdocs.pm/httpoison/1.8.0/HTTPoison.html#request/4